### PR TITLE
Fix TopMenu Popper#2545, Slider#2542, #2544

### LINKED ISSUE #2545: Menu Bar: Unexpected dropdown menu
There is a space in the "?" dropdown menu, which should not be there.

Screenshot from 2019-07-14 15-44-17

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2545, #2542, #2544 

Changes: 
- [x] Fix Menu Bar: Unexpected dropdown menu
- [x] Fix Admin Slider bug, alignment
- [x] Fix Slider in mobile view

Demo Link: https://pr-2549-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-07-14 at 10 07 32 PM
Screenshot 2019-07-14 at 10 07 41 PM
Screenshot 2019-07-14 at 10 08 04 PM

### CODE REVIEW

**Comment by fragm3** [comment]
@mariobehling except for table everything is fixed, we are having a limitation of using material UI here. Not possible to break the table into 2 lines,

**Comment by mariobehling** [comment]
Thank you!

Small changes, on the upload slider screen.
* Please add information on the expected image size, e.g. * Required image ratio is Height xxx px x Width xxx px"
* Make the "Save" button blue (according to color scheme)

**Comment by fragm3** [comment]
@mariobehling made the changes:
Screenshot 2019-07-15 at 3 36 56 AM

The button is actually blue, it's disabled by default if you enter appropriate data is becomes enabled.

File: src/components/cms/SkillSlideshow/index.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const settings = {
   speed: 500,
   nextArrow: <NextArrow />,
   prevArrow: <PrevArrow />,
-  autoplay: true,
+  // autoplay: true,
```
Comment (akshatnitd) [inline src/components/cms/SkillSlideshow/index.js]: Why have you commented it?

Comment (fragm3) [reply src/components/cms/SkillSlideshow/index.js]: Oh, left it by mistake

**Comment by fragm3** [comment]
@akshatnitd made the changes:

Screenshot 2019-07-15 at 9 57 48 AM

**Pull request merged**